platforms-serverless-vercel/vercel-with-redwood/api/package.json

```diff
@@ -7,7 +7,7 @@
     "@redwoodjs/graphql-server": "2.2.0"
   },
   "resolutions": {
-    "prisma": "3.8.0-dev.23",
-    "@prisma/client": "3.8.0-dev.23"
+    "prisma": "4.3.0-dev.8",
+    "@prisma/client": "4.3.0-dev.8"
```

Comment:
That was very outdated

Reply:
Seems we have nothing that bumps it automatically

Reply:
That's quite a problem then, seems this project is not working great.

Reply:
Best create a separate issue (if it does not exist yet)

Reply:
I wanted to create an issue, but it seems to work mostly as expected, see https://github.com/prisma/ecosystem-tests/issues/1921#issuecomment-1223814773

The patch script works (tried it), and updates the resolutions before testing.
But we do not commit this as we only commit during `check-for-update`